src/components/Home/index.js

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchLocationsData } from '../../redux/locations/locationsSlice';
+import LocationCard from '../LocationCard';
+
+import css from './Home.module.css';
+
+const Home = () => {
+  const { locations, locationsLoading } = useSelector((state) => state.locations);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (locations.length === 0) {
+      dispatch(fetchLocationsData());
+    }
+  }, [dispatch, locations.length]);
+
+  if (locationsLoading) {
```

Comment:
* ##OPTIONAL 
* I mark this as optional because the previous CR missed this required feature which is currently missing, I strongly suggest that you implement it so that your capstone project has all the required functionality [Check the requirement here ](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=one%20page%20with%20a%20list%20of%20items%20that%20could%20be%20filtered%20by%20some%20parameters%3B%20in%20the%20example%20below%2C%20it%27s%20a%20list%20of%20metrics%20that%20can%20be%20filtered%20by%20the%20country%20(imagine%20a%20search%20field%20to%20introduce%20the%20country%20name%20like%20Italy%2C%20Croatia%2C%20etc.).%20This%20page%20should%20be%20your%20homepage.)

> One page with a list of items that could be filtered by some parameters; in the example below, it's a list of metrics that can be filtered by the country (imagine a search field to introduce the country name like Italy, Croatia, etc.). This page should be your homepage.

Reply:
@Davidosky007 could you kindly expand on this? I didn't quite get it. I think the filtering was done by countries from  Asia in this project. I think this requirement is met